docs/src/main/tut/index.md

```diff
@@ -48,7 +105,14 @@ needs now, and easily port if and when your needs change.
 
 [blaze](http://github.com/http4s/blaze) is an NIO framework.  Run http4s on blaze for maximum throughput.
 
-{%code_ref ../../../examples/blaze/src/main/scala/com/example/http4s/blaze/BlazeExample.scala blaze_server_example %}
+```scala
```

Comment:
Tonight I learned that if you start a server in a tut shed, it starts a server.

That's fine.  We can show how to shut it down.